src/store/general/actions.ts

```diff
@@ -10,6 +10,10 @@ const actions: ActionTree<State, StateInterface> = {
       txHash,
     };
     commit('pushAlertMsg', alert);
+
+    setTimeout(() => {
+      commit('removeAlertMsg', 0);
+    }, 4000);
```

Comment:
What do you think about increasing timeout to 8 or 10 seconds, since there is a option to manually close toast? I think we should  give users more time to eg. copy transaction hash or similar.

Reply:
yes that makes sense. what do you think especially @Kahonnohak  ?

Reply:
Lets give it a go then! 8sec? I'm sorry I missed this comment